py_zipkin/encoding/_helpers.py

```diff
@@ -128,26 +122,22 @@ def build_v1_span(self):
         :return: newly generated _V1Span
         :rtype: _V1Span
         """
-        # We are simulating a full two-part span locally, so set cs=sr and ss=cr
-        full_annotations = OrderedDict(
-            [
-                ("cs", self.timestamp),
-                ("sr", self.timestamp),
-                ("ss", self.timestamp + self.duration),
-                ("cr", self.timestamp + self.duration),
-            ]
-        )
-
-        if self.kind != Kind.LOCAL:
-            # If kind is not LOCAL, then we only want client or
-            # server side annotations.
-            for ann in _DROP_ANNOTATIONS_BY_KIND[self.kind]:
-                del full_annotations[ann]
-
-        # Add user-defined annotations. We write them in full_annotations
+        annotations = OrderedDict([])
```

Comment:
@adriancole can you double check this is correct?

mr/ms spans have no 2nd annotation?